tests/orm/utils/test_managers.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Tests for the various node managers (.inputs, .outputs, .dict, ...)."""
+import pytest
+
+from aiida.backends.testbase import AiidaTestCase
+from aiida import orm
+from aiida.common import exceptions as exc
+from aiida.common import LinkType
+
+
+class TestAttributeManager(AiidaTestCase):
```

Comment:
Since (I believe) we want to slowly phase out the `AiidaTestCase`, could you try if this "just works" as a pure pytest test? Only if you feel like it, of course.

Reply:
Done. @ramirezfranciscof for me this PR is ready for a final review